docs/manual/manual_stock_led.rst

```diff
@@ -0,0 +1,314 @@
+            :height: 350
+
+            Vertiq 81-08 Gen 2 LED Connections
+
+The LED board's power connections should be tied to your module's V+ and V- ports ensuring that GND attaches to V-, and V+ connects to V+. The LED board has an operating range of 
+3-14S (11.1-52V nominal). Operating the LED board above this range can result in serious damage to the board. Operating the board below this range will simply prevent it from 
+operating as expected.
+
+The LED board uses a 4-pin JST-ZE connector in order to interface with your module. The following image displays the LED board's pinout.
+
+.. image:: ../_static/manual_images/leds/led_board_connections.png
+
+Please note that the connector's clip is at the top of this image. Take care to ensure that the module's RGB LED pad connects with the LED board's RGB LED pin, 
+and the module's white LED pad attaches to the LED board's white LED pin. Failure to do so can result in damage to your LED board as well as your module.
+
+Controlling Your LEDs
+========================
+.. warning::
+    Do not plug in your module's LED board while the module is already powered up. “Hot plugging” your module this way can cause damage to the LED board as well 
+    as the module itself. Always ensure that the LED board is fully connected with the module before powering it on.
+
+.. _default_pattern:
+
+Default Pattern
+******************
+
+By default, Vertiq modules control the RGB LED to display a static blue color, and the white LEDs to be shut off. On startup, the RGB LED should be illuminated.
+
+.. image:: ../_static/manual_images/leds/vertiblue_default_on.png
+    :width: 60%
+
+The default :ref:`strobing pattern <strobing_configuration>` for both the white LED and RGB led is to blink on and off 3 times, remain off, then restart the pattern with a 
+period of 2.5 seconds.
+
+.. raw:: html
+
+    <style type="text/css">
+    .center_vid {   margin-left: auto;
+                    margin-right: auto;
+                    display: block;
+                    width: 75%; 
+                }
+    </style>
+    <video class='center_vid' controls muted><source src="../_static/manual_images/leds/default_blinking.mp4" type="video/mp4"></video>
+
+RGB LED Configuration
+***********************
+Your module has control over the RGB LED's red, green, and blue intensities as well as its strobing output. Strobing configuration is identical for both the RGB and 
+white LEDs, and is covered in :ref:`strobing_configuration`. By default, strobing is disabled for your RGB LED. RGB LED control parameters are found in 
+the :ref:`RGB LED client reference table <rgb_led_message_table>`.
+
+The pertinent control parameters for RGB LED control are ``red``, ``green``, ``blue``, and ``update_color``. Each of the RGB parameters can be set, gotten, and 
+saved, and can be an integer [0, 255] where 255 indicates the maximum intensity. ``update_color`` is a trigger mechanism that causes your module to command the LED 
```

Comment:
We list out 4 parameters as RGB LED parameters, including update_color, then we say "Each of the RGB parameters can be....an integer [0,255]". Possibly a bit misleading for that sentence that update color is also settable to 255, maybe better to say " 'red', 'green', and 'blue' can be....an integer [0,255]", for maximum specificity